polaris-core/src/main/java/org/apache/polaris/core/storage/PolarisStorageConfigurationInfo.java

```diff
@@ -240,19 +241,23 @@ public void validateMaxAllowedLocations(int maxAllowedLocations) {
   /** Polaris' storage type, each has a fixed prefix for its location */
   public enum StorageType {
     S3("s3://"),
-    AZURE("abfs"), // abfs or abfss
+    AZURE(List.of("abfs", "wasb", "abfss", "wasbs")),
```

Comment:
Since we're filling out the whole list let's also make these `abfs://`, `wasb://`, `abfss://` and `wasbs://` for consistency with the others. It's a bit of messy right now that we relied on String.startsWith here such that `abfsasdfasdf://` is possibly interpreted as a valid scheme.